src/app/components/Progress/Progress.module.css

```diff
@@ -0,0 +1,33 @@
+.progress {
+  display: grid;
+  place-items: center;
+  height: 3rem;
+  background-color: #dfdfdf;
+  border-radius: 1rem;
+}
+
+progress {
```

Comment:
Be careful using element selectors. This effects all `progress` in your project.

Reply:
Changed it to the className